lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -323,96 +389,109 @@ class FatMemManager : public OpSemMemManager {
   FatMemValTy MemSet(PtrTy ptr, Expr _val, unsigned len, FatMemValTy mem,
                      uint32_t align) override {
     return mkFatMem(
-        m_mem.MemSet(mkRawPtr(ptr), _val, len, mkRawMem(mem), align));
+        m_main.MemSet(mkRawPtr(ptr), _val, len, mkRawMem(mem), align),
+        m_main.MemSet(mkRawPtr(ptr), _val, len, mkSlot0Mem(mem), align),
+        m_main.MemSet(mkRawPtr(ptr), _val, len, mkSlot1Mem(mem), align));
   }
 
   /// \brief Executes symbolic memcpy with concrete length
   FatMemValTy MemCpy(PtrTy dPtr, PtrTy sPtr, unsigned len, Expr memTrsfrRead,
                      uint32_t align) override {
-    return mkFatMem(m_mem.MemCpy(mkRawPtr(dPtr), mkRawPtr(sPtr), len,
-                                 mkRawMem(memTrsfrRead), align));
+    return mkFatMem(m_main.MemCpy(mkRawPtr(dPtr), mkRawPtr(sPtr), len,
```

Comment:
I think is is ok. Would be good to document the intended semantics. Some examples with `MemCpy` will help narrow this down.

I think we now assume that every byte of raw memory has slot0 and slot1, which themselves are usually wider. `memcpy` copies bytes and underlying slots. 

Seems like good semantics.